test/support/apps/example/bin/test

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+mix deps.get || { echo 'Dependencies could not be fetched!'; exit 1; }
+MIX_ENV=test mix ecto.drop || { echo 'Database could not be dropped'; exit 1; }
+MIX_ENV=test mix torch.install || { echo 'Torch could not be installed!'; exit 1; }
+MIX_ENV=test mix torch.gen.html Blog Post posts title:string || { echo 'Torch files not generated!'; exit 1; }
```

Comment:
I think we should include a more detailed schema generated here including all the different field types.